src/components/base/SideBar.vue

```diff
@@ -0,0 +1,192 @@
+<template>
```

Comment:
Sidebar bug when the screen is small?

<img width="500" alt="Screen Shot 2022-01-05 at 11 55 29 AM" src="https://user-images.githubusercontent.com/5270855/148269954-89eab7bb-e283-4678-bd4b-1ca9fb6cce6f.png">

Reply:
fixed!